main.js

```diff
@@ -5,10 +5,17 @@ const map = new Array();
 var playerLocation = {
     floor: 0,
     row: 0,
-    col: 4
-}; // row, colume
+    col: 4,
+    health: 100,
+    attack: getRandomInt(10) + 10,
```

Comment:
start player with more attack but less defense 
attack: getRandomInt(10) +5,
defense: getRandomInt(5) +5,

Reply:
the player shoul;d be able to choose own stats at the begining of the game and not be stuck with set stats 
/the abillity to add certain stat points based off character choice in beginning 
/have a home page or pop up to choose character in beginning.